### PR TITLE
refactor: Include the admin client via ember-browserify

### PR DESCRIPTION
**Pull request by courajs**
This is much nicer than manually copying over into /vendor, and using a
global

### CODE REVIEW

File: app/services/hoodie-account-admin.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-/* global AccountAdmin */
 import Ember from 'ember';
+import HoodieAdmin from 'npm:@hoodie/admin-client';
```
Comment (gr2m) [inline app/services/hoodie-account-admin.js:2]: 👏

File: app/services/hoodie-account-admin.js [hunk 2]
```diff
@@ -8,7 +8,8 @@ const {
 export default Ember.Service.extend({
   init() {
     this._super(...arguments);
-    this.admin = new AccountAdmin({url: '/hoodie/account/api'});
+    this.admin = new HoodieAdmin();
+    window.admin = this.admin;
```
Comment (gr2m) [inline app/services/hoodie-account-admin.js]: is window.admin for debugging? I like it, but maybe add a comment?

Comment (courajs) [inline app/services/hoodie-account-admin.js]: Actually left in by accident, but it will be useful. I'll add a comment

**Comment by gr2m** [comment]
LGTM 👍

**Comment by courajs** [comment]
@gr2m updated with comment

**Comment by gr2m** [comment]
purrrfact 👍

**Pull request merged**